mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -97,6 +98,7 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
     case "recall" => true
     case "weightedPrecision" => true
     case "weightedRecall" => true
+    case "accuracy" => true
```

Comment:
I think one big case could be better, which leaves an incremental option of returning false in the future. If we simple return true now and any metric in the future returns false, we need to add back everything else returning true, which has the potential of missing some metric and is less obvious for people making changes.